hackathon/api/views.py

```diff
@@ -66,3 +68,30 @@ class HackathonEventAPIView(generics.ListAPIView):
 
     def get_queryset(self):
         return HackathonEvent.objects.all()
+
+
+class ProblemCategoryClass(HttpResponse):
+    serializer_class = ProblemCategorySerializer
+
+    def get_problem_category(request):
+        if request.method == 'GET':
+            category = ProblemCategory.objects.all().values('id','category','description','created_at')
+            # serializer = ProblemCategorySerializer(category,many=True)
+            listed = list(category)
+            return JsonResponse(listed, safe=False)
+        else:
+            print("Only GET is allowed")
+
+    def get_problem_statements(request, category_id):
```

Comment:
Should be separate class API

Reply:
okay, 
i though both had the same parent i.e category hence, will discuss these things before hand next time ,
thanks